### ISSUE TITLE
Allow filtering devices by hostname

### ISSUE THREAD

**Issue by mt-reynolds** [open-amt-cloud-toolkit/mps#548]
On the API GET /devices can you add a query param for the hostname.

**Comment by gbinder2** [comment]
I agree with the request for enhancement.  Given that some companies consider hostname PII, I believe we should work to find some protections for this field.  If the field is a FQDN and the admin is searching for the first part only, this could be more complex.  If the field is hostname only, their might be more options.
